dht-node/src/dht_node/index.ts

```diff
@@ -184,3 +187,38 @@ export const startDHT = async (topic: string): Promise<void> => {
     logger.error('DHT unexpected error:', err)
   }
 }
+
+async function writeHomeCommunityEnries(pubKey: any): Promise<CommunityApi[]> {
+  const homeApiVersions: CommunityApi[] = Object.values(ApiVersionType).map(function (apiEnum) {
+    const port =
+      Number.parseInt(CONFIG.FEDERATION_COMMUNITY_API_PORT) + Number(apiEnum.replace('_', ''))
```

Comment:
I don't understand why you want to add the API version to the port! Or is it the concept that we call different GraphQL endpoints for different API versions?

Reply:
So for version 1_0 we would get port 5010? 
This seems not right for me! Why don't we use the version in the endpoint url instead of making some magic on ports

Reply:
Cause with this we would not use the structure you used in the Test where you showed that we can have different calls per version!
In my opinion the endpoint url should resemble to this:
- http://localhost:5000/api/v1_0
- http://localhost:5000/api/v2_0
- http://localhost:5000/api/1_0
- http://localhost:5000/api/2_0

Reply:
this implementation mirrors the requirements and results from our last meeting about this topic.
the pattern for an graphql-endpoint looks like:
<host>:<port>/api/<apiversion>/

<apiversion> = no leading 'V', only numbers and underscore, <major-version>_<minor-version>

apiversion should mirror the port-number

the port follows the rules:
- baseport of federation is 5000
- till 5009 is reserverd for DHT
- version V1_0 => 5010
- version V1_1 => 5011
- version V2_0 => 5020
- ...

Reply:
> the port follows the rules:
>
> - baseport of federation is 5000
> - till 5009 is reserverd for DHT
> - version V1_0 => 5010
> - version V1_1 => 5011
> - version V2_0 => 5020
> - ...

OK I understand what you are saying, I think that it will be difficult to setup every endpoint.

The test project was with one endpoint that would be called with the following url:
- 1_0: ``https://localhost:5000/api/1_0``
- 1_1: ``https://localhost:5000/api/1_1``
- 2_0: ``https://localhost:5000/api/2_0``
But now you want to initialize them for each version, the unit test say that it would give this urls:
- 1_0: ``https://localhost:5001/api/``
- 1_1: ``https://localhost:5002/api/``
- 2_0: ``https://localhost:5003/api/``

Reply:
yes that's correct, the testdata will not exact match the data, which the dht-node will write in database.
But all the TestResolvers for all three apiversions will be removed with the next federation implementation step. 
They where only implemented to ensure a technical solution for a versionized graphql-endpoint, because graphql for itself will not support versioning of endpoints.
So on my opinion i will not change these unit tests anymore for code, which will be removed. Can you agree?